ophyd/control_layer.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+__all__ = ['setup',
+           'caput', 'caget',
+           'get_pv',
+           'pv_form',
+           'thread_class']
+
+if os.environ.get('CAPROTO', False):
```

Comment:
Re-upping @stuartcampbell's comment: something more specific, like:

```python
if os.environ.get('OPHYD_CONTROL_LAYER').lower() == 'caproto':
```

Reply:
I like it, much better than my suggestions.

Reply:
and that gives a nice hook to a future extension where `ophyd.setup(...)` gives you a code-level (rather than ENV level) hook to control this.